src/index.js

```diff
@@ -50,15 +23,12 @@ const start = async (component) => {
       throw ex
     }
   }
-
   // 3.b Write file
   const fullpath = path.resolve(output.dir, output.file)
```

Comment:
This is the `index.jsx`, I don't think we need it anywhere else but in this script